src/types/FilterMode.ts

```diff
@@ -0,0 +1 @@
+export type FilterMode = 'activ' | 'completed' | 'all';
```

Comment:
```suggestion
export type FilterMode = 'active' | 'completed' | 'all';
```

Reply:
better to create some enum for filter values

Reply:
Not fixed